### PR TITLE
fix: clarify OSS to OSS, Cloud, Enterprise

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Closes #https://github.com/influxdata/docs-v2/issues/4534

- The v2.4 and cloud documents both guide the user on how to replicate data from OSS to Cloud so i've removed the note on the index page with the link out.
- Updated the header and description to clarify the guide can be used to replicate data from OSS to OSS, Cloud, or enterprise
- Added a note within the guide to clarify the guide can be used to replicate data from OSS to OSS, Cloud, or enterprise by adjusting the remote connection values to the desired instance 
- Both v2.4 and cloud have the same titles, headers and descriptions 

image

image

### CODE REVIEW

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 1]
```diff
@@ -16,16 +16,26 @@ related:
...
-{{% oss-only %}}InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.{{% /oss-only %}}
-{{% cloud-only %}}InfluxDB Cloud instance.{{% /cloud-only %}}
+the incoming data of select buckets to one or more buckets on a remote 
+InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.
+
+The following guide demonstrates how to replicate data from InfluxDB OSS to 
+InfluxDB Cloud and can also be used to replicate data to the following instances:
+- InfluxDB OSS to InfluxDB OSS
+- InfluxDB OSS to InfluxDB Cloud
+- InfluxDB OSS to InfluxDB Enterprise
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:26]:
```suggestion
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Enterprise
```

or something like this, b/c you already have the the note for OSS and Enterprise:

The following guide shows how to replicate data for the following use cases:
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Cloud
- InfluxDB OSS to InfluxDB Enterprise
...
Configure remote connection for the InfluxDB instance that you want to replicate data to.

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Replicate data from a remote instance to InfluxDB Cloud
+title: Replicate data from InfluxDB OSS
 weight: 106
 description: >
   Use InfluxDB Edge Data Replication to replicate the incoming data of select
-  buckets to one or more buckets on a remote InfluxDB OSS, InfluxDB Cloud, or
-  InfluxDB Enterprise instance.
+  buckets from InfluxDB OSS to one or more buckets in either InfluxDB Cloud, 
+  InfluxDB OSS, or InfluxDB Enterprise instances.
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:7]:
```suggestion
  buckets from InfluxDB OSS to buckets in remote InfluxDB Cloud,
  InfluxDB OSS, or InfluxDB Enterprise instances.
```

**Review by jstirnaman — APPROVED** [review]
A couple suggestions. Otherwise, looks good.

File: content/influxdb/cloud/write-data/replication/replicate-data.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 ---
-title: Replicate data from InfluxDB OSS to InfluxDB Cloud
+title: Replicate data from InfluxDB OSS
 weight: 201
 description: >
-  Use replication streams to replicate the incoming data of select buckets to one or more buckets on a remote InfluxDB instance.
-
+  Use InfluxDB Edge Data Replication to replicate the incoming data of select
+  buckets from InfluxDB OSS to buckets in remote InfluxDB Cloud, InfluxDB OSS, or 
+  InfluxDB Enterprise instances.
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/replication/replicate-data.md:7]:
```suggestion
Replicate data from select InfluxDB OSS buckets to remote buckets in InfluxDB Cloud, InfluxDB OSS, or InfluxDB Enterprise instances.
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Replicate data from a remote instance to InfluxDB Cloud
+title: Replicate data from InfluxDB OSS
 weight: 106
 description: >
   Use InfluxDB Edge Data Replication to replicate the incoming data of select
```
Comment (kelseiv) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:5]:
```suggestion
Replicate data from select InfluxDB OSS buckets to remote buckets in InfluxDB Cloud, InfluxDB OSS, or InfluxDB Enterprise instances.
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-  buckets to one or more buckets on a remote InfluxDB OSS, InfluxDB Cloud, or
-  InfluxDB Enterprise instance.
+  buckets from InfluxDB OSS to buckets in remote InfluxDB Cloud, InfluxDB OSS, or 
+  InfluxDB Enterprise instances.
```
Comment (kelseiv) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:7]:
```suggestion
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 6]
```diff
@@ -16,16 +16,25 @@ related:
...
+- InfluxDB OSS to InfluxDB Enterprise
 
 ## Configure a replication stream
 
 Use the [`influx` CLI](/influxdb/v2.4/tools/influx-cli/) or the
 [InfluxDB {{< current-version >}} API](/influxdb/v2.4/reference/api/) to configure
 a replication stream.
 
+{{% note %}}
+To replicate data to InfluxDB OSS or InfluxDB Enterprise, adjust the desired
```
Comment (kelseiv) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:34]:
```suggestion
To replicate data to InfluxDB OSS or InfluxDB Enterprise, adjust the 
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 7]
```diff
@@ -16,16 +16,25 @@ related:
...
-the incoming data of select buckets to one or more buckets on a remote
-{{% oss-only %}}InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.{{% /oss-only %}}
-{{% cloud-only %}}InfluxDB Cloud instance.{{% /cloud-only %}}
+the incoming data of select buckets to one or more buckets on a remote 
+InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.
+
+The following guide demonstrates how to replicate data from InfluxDB OSS to 
+InfluxDB Cloud and can also be used to replicate data to the following instances:
+- InfluxDB OSS to InfluxDB OSS
+- InfluxDB OSS to InfluxDB Enterprise
```
Comment (kelseiv) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:25]:
```suggestion
Replicate data from InfluxDB OSS to InfluxDB Cloud, InfluxDB OSS, or InfluxDB Enterprise.
```

**Review by kelseiv — APPROVED** [review]

**Pull request merged**